src/main/frontend/app/routes/studio/canvas/nodetypes/frank-node.tsx

```diff
@@ -19,16 +19,11 @@ import variables from '../../../../../environment/environment'
 import { useSettingsStore } from '~/routes/settings/settings-store'
 import HandleMenu from './components/handle-menu'
 import type { ActionType } from './components/action-types'
+import { ChildNode } from './child-node'
+import { findChildRecursive } from '~/stores/child-utilities'
+import { canAcceptChildStatic } from './node-utilities'
 
-export interface ChildNode {
-  id: string
-  subtype: string
-  type: string
-  name?: string
-  attributes?: Record<string, string>
-}
-
-export type FrankNode = Node<{
+export type frankNode = Node<{
```

Comment:
Hmm maybe calling it `FrankNodeType` would be better than putting the name into camel-case